ledger/participant-integration-api/src/main/resources/db/migration/oracle/V100_0__Append_only_schema.sql

```diff
@@ -0,0 +1,485 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
FYI: this needs to go to oracle-appendonly directory instead
This can be incremental to the legacy schema like with Postgres, or from scratch like with H2. I built the tooling to make this possible at FlywayMigrations:85